bin/prefix-vendor-namespace.php

```diff
@@ -110,6 +124,43 @@ function( $file ) use ( $namespace, $new_namespace ) {
 	);
 }
 
+if ( count( $file_notices ) ) {
+	printf(
+		'Several files were modified without changes to namespace: %s' . PHP_EOL,
+		implode('; ', $file_notices)
```

Comment:
This file does have a phpcs ignore but it would be nice to stick to the default WP spacing rules for easy legibility.
```suggestion
		implode( '; ', $file_notices )
```